cognite/neat/graph/loaders/_rdf2dms.py

```diff
@@ -0,0 +1,191 @@
+import json
+import warnings
+from collections import defaultdict
+from collections.abc import Iterable
+from pathlib import Path
+from typing import Any, cast
+
+import yaml
+from cognite.client import CogniteClient
+from cognite.client import data_modeling as dm
+from cognite.client.data_classes.data_modeling import ViewId
+from pydantic import HttpUrl, TypeAdapter, ValidationError, ValidationInfo, create_model, field_validator
+from pydantic.main import Model
+from rdflib.query import ResultRow
+from rdflib.term import URIRef
+
+from cognite.neat.graph.stores import NeatGraphStoreBase
+from cognite.neat.rules.issues import NeatValidationError
+from cognite.neat.rules.models import DMSRules
+from cognite.neat.rules.models.data_types import _DATA_TYPE_BY_DMS_TYPE
+
+from ._base import CDFLoader
+
+
+class DMSLoader(CDFLoader[dm.InstanceApply]):
+    def __init__(
+        self,
+        graph_store: NeatGraphStoreBase,
+        data_model: dm.DataModel[dm.View],
```

Comment:
This would work if underlaying graph's properties and classes are transformed such that:
Class -> View
properties -> view.property_ids

Reply:
and that class_id and property_id are in the same namespace, however RDF is often composed of properties and classes which originate from various namespace. 

Two options:
1. transform original/source graph to same namespace, such that class_id and property_id contain only DMS allowed characters (allow Select queries afterwards)
2. transform graph on query, i.e. use construct query to create "virtual graphs"

Reply:
As discussed we assume same namespace